experiments/calibration/calibrate_land.jl

```diff
@@ -17,37 +31,46 @@ rng_seed = 2
 rng = Random.MersenneTwister(rng_seed)
 FT = Float64
 
-# Calibrate the land model with:
-const variable_list = ["swu"] # variables you want to capture by adjusting your priors
-const n_iterations = 10 # 1 iterations takes ~ 1.5 hour with current settings (resolution, 2 year simulation)
-const spinup_period = Year(1)
-# potentially we could add time_for_calibration (currently 1 year)
-
 # Using the following priors:
-include(joinpath(@__DIR__, "priors.jl"))
+include(joinpath(@__DIR__, string("priors_", model, ".jl")))
```

Comment:
should this be moved up?

Reply:
is this how you control which parameters are being calibrated?

Reply:
yes, the priors file (priors_land.jl of priors_bucket.jl) contains the parameters to calibrate and their distribution. Why do you want to move it up? We can discuss

Reply:
mainly because this is something a user will change in between runs, and there was a comment in the code about how things the user changes are above this section